build.sbt

```diff
@@ -715,6 +715,24 @@ lazy val kernelDefaults = (project in file("kernel/kernel-defaults"))
     unidocSourceFilePatterns += SourceFilePattern("io/delta/kernel/"),
   ).configureUnidoc(docTitle = "Delta Kernel Defaults")
 
+
+lazy val sparkKernelDsv2 = (project in file("spark-dsv2"))
```

Comment:
Should we add it to `sparkGroup`? I think this might automatically add it to CI. If not, will this be added to CI later?